src/client/styles/main.scss

```diff
@@ -174,3 +174,12 @@ select {
     background: #fff;
   }
 }
+
+// For now until it's in the Design Toolkit
+.nypl-results-publication {
+  display: inline-block;
+  margin-right: 1rem;
+  color: #080807;
+  font-size: "";
```

Comment:
What's the significance of the empty string here? Is this for showing/hiding the DOM element?

Reply:
Good question... I copied it from the design toolkit. Maybe @ricardoom has a better (an actual) answer?